maybe/syscall_filters.py

```diff
@@ -108,6 +109,29 @@ def substitute_open(path, flags):
         return None
 
 
+def format_mknod(path, type):
+    path = abspath(path)
+    if exists(path):
+        return None
+    elif (type & S_IFREG):
+        return "%s %s" % (T.cyan("create file"), T.underline(path))
+    elif (type & S_IFIFO):
+        return "%s %s" % (T.cyan("create named pipe"), T.underline(path))
+    else:
+        # TODO: add support for block and char special files
```

Comment:
Could you add these too? The last remaining case, Unix domain sockets, seem to leave a persistent inode also.